File: bug-fixes/buggy-code-part-6.js
function removeNumbers(str) {
  // Or: str.replace(/\d/g,'')

  return str.split('')
		.filter(c => isNaN(c))
		.join('')
}

removeNumbers("mubashir1") // "mubashir"
removeNumbers("12ma23tt") // "matt"
removeNumbers("e1d2a3b4i5t6") // "edabit"
removeNumbers("pakistan007") // "pakistan"
removeNumbers("ai4653rf53or4235ce") // "airforce"

File: bug-fixes/buggy-code-part-6.py
def remove_numbers(string):
    return "".join(i for i in string if i.isalpha())
  
remove_numbers("foo1"), "foo"
remove_numbers("12ma23tt"), "matt"
remove_numbers("e1d2a3b4i5t6"), "edabit"
remove_numbers("czech007"), "czech"
remove_numbers("ai4653rf53or4235ce"), "airforce"